packages/core/src/core/fetch.ts

```diff
@@ -448,6 +448,9 @@ const fetchAndProcessMergeErrors = async (
             withChangesDetection,
           })
           const { updatedConfig, errors } = fetchResult
+          if (updatedConfig !== undefined) {
+            log.debug(`Salto got config suggestions in ${accountName} account: ${updatedConfig.message}`)
```

Comment:
nit: as we're logging the reasons for the config update, maybe write something similar to this ? 
```suggestion
            log.debug(`In account: ${accountName}, received config suggestions for the following reasons: ${updatedConfig.message}`)
```

Reply:
I Didn't see the nit comment and already merged. I will change it on Sunday and ask for PR again.